### PR TITLE
Remove appcompat dependency

### PR DESCRIPTION
**Pull request by JDSM01**
**What does this PR do?**

Removes appcompat dependency as it was no longer being used

**Database changed?**

No

**Where should the reviewer start?**

- [ ] Foo.py
- [ ] Bar.py

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APPC-XXXX](https://aptoide.atlassian.net/browse/APPC-<Jira ticket number>)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-XXXX](https://aptoide.atlassian.net/browse/APPC-<Jira ticket number>)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 

**Code Review Checklist**

- [ ] Architecture
- [ ] Documentation on public interfaces
- [ ] Database changed?
- [ ] If yes - Database migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] Functional QA tests pass

### CODE REVIEW

File: android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/InstallDialogActivity.java [hunk 1]
```diff
@@ -54,7 +54,7 @@
   private static String installButtonTextColor = "#ffffffff";
   private final String GOOGLE_PLAY_URL =
       "https://play.google.com/store/apps/details?id=" + BuildConfig.BDS_WALLET_PACKAGE_NAME;
-  private final String CAFE_BAZAAR_URL = "https://cafebazaar.ir/app/com.hezardastaan.wallet";
+  private final String CAFE_BAZAAR_URL = "bazaar://details?id=com.hezardastan.wallet";
```
Comment (brunovarelaAptoide) [inline android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/InstallDialogActivity.java:57]: 2 different urls ?

File: android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/CafeBazaarResponseAsync.java [hunk 2]
```diff
@@ -38,9 +37,13 @@
...
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+      try (OutputStream os = huc.getOutputStream()) {
+        byte[] input = jsonInputString.getBytes(StandardCharsets.UTF_8);
+        os.write(input, 0, input.length);
+      }
+    } else {
+      return 404;
     }
     huc.connect();
     int responseCode = 404;
```
Comment (brunovarelaAptoide) [inline android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/CafeBazaarResponseAsync.java]: Isn't this function always returning 404 ?

File: android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/CafeBazaarResponseAsync.java [hunk 3]
```diff
@@ -38,9 +37,13 @@
         + "\t\t}\n"
         + "\t}\n"
         + "}";
-    try (OutputStream os = huc.getOutputStream()) {
-      byte[] input = jsonInputString.getBytes(StandardCharsets.UTF_8);
-      os.write(input, 0, input.length);
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+      try (OutputStream os = huc.getOutputStream()) {
```
Comment (brunovarelaAptoide) [inline android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/CafeBazaarResponseAsync.java]: Should this try clause have a catch?

File: android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/CafeBazaarResponseAsync.java [hunk 4]
```diff
@@ -17,7 +16,7 @@
     this.responseListener = responseListener;
   }
 
-  @RequiresApi(api = Build.VERSION_CODES.KITKAT) private int getResponseCode() throws IOException {
```
Comment (brunovarelaAptoide) [inline android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/CafeBazaarResponseAsync.java:20]: should this method return a IOException?

**Review by brunovarelaAptoide — CHANGES_REQUESTED** [review]

File: android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/InstallDialogActivity.java (hunk 1 above)
Comment (JDSM01) [reply android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/InstallDialogActivity.java:57]: There was, this PR is to put the same one in both flows

**Pull request merged**